include/target/impl/flash_get_config_from_rom_old.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ */
+
+#pragma once
+
+#include <target/flash.h>
+
+/* ROM's flash config data for some old chips */
+extern esp_rom_spiflash_chip_t g_rom_flashchip;
+
+inline static
+const esp_rom_spiflash_chip_t *flash_impl_get_config_from_rom_old(void)
```

Comment:
Do we really need separate headers for this functions?
How about have it like macro and keep it with `flash_impl_get_config_from_rom` and other similar ROM related staff in one header?